src/sentry/static/sentry/app/components/events/eventEntries.tsx

```diff
@@ -243,9 +262,11 @@ class EventEntries extends React.Component {
             location={location}
           />
         )}
-        {!objectIsEmpty(event.sdk) && <EventSdk sdk={event.sdk} />}
-        {!isShare && event?.sdkUpdates?.length > 0 && <EventSdkUpdates event={event} />}
-        {!isShare && event.groupID && (
+        {event?.sdk && !objectIsEmpty(event.sdk) && <EventSdk sdk={event.sdk} />}
+        {!isShare && event?.sdkUpdates && event.sdkUpdates.length > 0 && (
```

Comment:
```suggestion
        {!isShare && event?.sdkUpdates?.length > 0 && (
```

Reply:
That doesn't work, then we compare undefined to 0.

Reply:
in the console if I type` undefined > 0 `I get `false`... but you can also leave as it is :)

Reply:
ok but typescript fails there

Reply:
true..good catch!!